Doc/whatsnew/3.14.rst

```diff
@@ -70,6 +70,78 @@ Summary -- Release highlights
 New Features
 ============
 
+.. _whatsnew-314-pep649:
+
+PEP 649: Deferred Evaluation of Annotations
+-------------------------------------------
+
+The :term:`annotations <annotation>` on functions, classes, and modules are no
+longer evaluated eagerly. Instead, annotations are stored in special-purpose
+:term:`annotate functions <annotate function>` and evaluated only when
+necessary. This is specified in :pep:`649` and :pep:`749`.
+
+The new :mod:`annotationlib` module provides tools for inspecting deferred
+annotations. Annotations may be evaluated in the :attr:`~annotationlib.Format.VALUE`
+format (which evaluates annotations to runtime values, similar to the behavior in
+earlier Python versions), the :attr:`~annotationlib.Format.FORWARDREF` format
+(which replaces undefined names with special markers), and the
+:attr:`~annotationlib.Format.SOURCE` format (which returns annotations as strings).
+
+This example shows how these formats behave::
+
+   from annotationlib import get_annotations, Format
+
+   def func(arg: Undefined):
+       pass
+
+   print(get_annotations(func, format=Format.VALUE))  # raises NameError
+   print(get_annotations(func, format=Format.FORWARDREF))  # {'arg': ForwardRef('Undefined')}
+   print(get_annotations(func, format=Format.SOURCE))  # {'arg': 'Undefined'}
+
+Implications if you write annotations in your code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you write annotations in your code (for example, for use with a static type
+checker), then this change probably does not affect you: you can keep
+writing annotations the same way you did in previous versions of Python.
+
+You will likely be able to remove quoted strings in annotations, which are frequently
+used for forward references. Similarly, if you use ``from __future__ import annotations``
+to avoid having to write strings in annotations, you may well be able to
+remove it. However, if you rely on third-party libraries that read annotations,
+those libraries may need changes to support unquoted annotations.
+
+Implications if you read ``__annotations__``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If your code reads the ``__annotations__`` attribute on objects, you may want
+to make changes in order to support code that relies on deferred evaluation of
+annotations. For example, you may want to use the :func:`annotationlib.get_annotations`
```

Comment:
"you may want to use the `annotations.get_annotations` with the `FORWARDREF` format" isn't quite grammatical:

```suggestion
annotations. For example, you may want to use :func:`annotationlib.get_annotations`
```